kymatio/scattering3d/tests/test_torch_scattering3d.py

```diff
@@ -85,54 +95,36 @@ def test_cdgmm3d(device, backend, inplace):
         if inplace:
             assert (x - z).norm().cpu().item() < 1e-7
 
-        with pytest.warns(UserWarning) as record:
+        with pytest.raises(RuntimeError) as record:
             x = torch.randn((3, 4, 3, 2), device=device)
             x = x[:, 0:3, ...]
             y = torch.randn((3, 3, 3, 2), device=device)
             backend.cdgmm3d(x, y)
-        assert "A is converted" in record[0].message.args[0]
+        assert "contiguous" in record.value.args[0]
 
-        with pytest.warns(UserWarning) as record:
+        with pytest.raises(RuntimeError) as record:
             x = torch.randn((3, 3, 3, 2), device=device)
             y = torch.randn((3, 4, 3, 2), device=device)
             y = y[:, 0:3, ...]
             backend.cdgmm3d(x, y)
-        assert "B is converted" in record[0].message.args[0]
+        assert "contiguous" in record.value.args[0]
 
         with pytest.raises(RuntimeError) as record:
             x = torch.randn((3, 3, 3, 2), device=device)
             y = torch.randn((4, 4, 4, 2), device=device)
             backend.cdgmm3d(x, y)
         assert "not compatible" in record.value.args[0]
 
-        x = torch.randn((2, 3, 3, 3, 2), device=device)
-        y = torch.randn((3, 3, 3, 2), device=device)
-        backend.cdgmm3d(x, y)
```

Comment:
Why was this here in the first place like this?

Reply:
I don't understand the question. Why shouldn't it have been here?

Reply:
Its not testing anything, nor was any of it used in a test.

Reply:
Looks like it's testing that broadcasting works properly? Just because there's no assert doesn't mean that it's not testing a certain code path (more like a smoke test than a standard unit test).